test/fixtures/line/labelsScriptableOptions.js

```diff
@@ -16,8 +16,10 @@ module.exports = {
       },
       plugins: {
         annotation: {
-          drawTime() {
-            return 'afterDatasetsDraw';
+          common: {
+            drawTime() {
+              return 'afterDatasetsDraw';
+            }
```

Comment:
Isn't the old syntax correct, because when I try that it works and with common it doesnt. Also cant find any reference to common in the source code

Reply:
fixed, will add the changes to the other pr, which is a breaking one.